src/components/molecules/Schedule/Schedule.tsx

```diff
@@ -66,10 +67,15 @@ export const Schedule: React.FC<ScheduleProps> = ({
     [scheduleStartHour, scheduleDate]
   );
 
+  const sortedEvents = useMemo(
```

Comment:
There shouldn’t be any need to separate this out into a new memo like this is there? Can we not just use .sort directly at the source of locatedEvents? (After refactoring the sort function as suggested in this review)